### ISSUE TITLE
Unclear instructions

### ISSUE THREAD

**Issue by jimmyfm** [crgeary/wp-jamstack-deployments#31]
The plugin is clear on how to use and where to obtain all the info to set it up.

I am unclear on what do I need to do on the netlify side to make publishing work as expected.

If you give me some pointers I can amend the plugin readme and make pull request.

**Comment by crgeary** [comment]
Hey,

There is nothing **specific to this plugin** that you need to do on Netlify's side.

Can you let me know what problem you're running into? Perhaps then I can better answer your question.

**Comment by jimmyfm** [comment]
Netlify requires a git repo, so I created a "hello world" one, created a new site based on that, did all the configuration of the WP plugin.
It seems working, but every time it redeploys the same "hello world" from the git repo. 
That's what makes me thing that some code is needed in that git repo.

Edit: Thanks for the incredibly quick response :)

**Comment by crgeary** [comment]
Ah.

All this plugin will do is tell Netlify that content has changed, and that it should rebuild the site. It doesn't provide Netlify with any of the content from the WordPress site.

You're correct, there is code needed in that repo. Unfortunately, it's quite a lot of code, and it's not something I can cover for you in a Github issue.

There are a number of ways to pull content from WordPress into Netlify. You can use an off the shelf static site generator, which has support for WordPress, such as [Gatsby](https://www.gatsbyjs.com/), or you can write your own scripts in most major languages (there's a [Node.js](http://wp-api.org/node-wpapi/) package to help with this).

There are LOTS of ways to go about this, but if you go the WordPress/Gatsby route there's a [tutorial on the Gatsby site](https://www.gatsbyjs.org/tutorial/wordpress-source-plugin-tutorial/) you can take a look at.

I'll keep this issue open, incase you have any other questions about it.

**Comment by jimmyfm** [comment]
It is way more complex than I have anticipated. 
At the beginning it was unclear to me that the plugin only triggers a build on Netlify, and a bunch of code needed to pull the data and style the site is also needed on that side.
Digging deeper on Gatsby site I found this [extremely detailed and useful three part tutorial](https://www.gatsbyjs.org/blog/2019-04-26-how-to-build-a-blog-with-wordpress-and-gatsby-part-1/).
Thank you for your input and direction in this journey, I will make sure to document my work and publish the code for my site.
